### PR TITLE
add ogb

### PR DESCRIPTION
**Pull request by hadim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/ogb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  sha256: fb2c67665489477e8ec5b1f858393d724dc9185c3e568c4bc042993d4c18b679
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - outdated
```
Comment (scopatz) [inline recipes/ogb/meta.yaml:20]: Is this really needed in host?

File: recipes/tkwant/bld.bat [hunk 2]
```diff
@@ -1 +1 @@
-%PYTHON% setup.py install --single-version-externally-managed --record=record.txt
+%PYTHON% setup.py install --single-version-externally-managed --record=record.txt
```
Comment (scopatz) [inline recipes/tkwant/bld.bat:1]: This file shouldn't be touched, but I was seeing this issue locally as well so :shrug:

File: recipes/ogb/meta.yaml (hunk 1 above)
Comment (hadim) [reply recipes/ogb/meta.yaml:20]: `grayskull` added it. I'll try to remove it.

File: recipes/tkwant/bld.bat (hunk 2 above)
Comment (hadim) [reply recipes/tkwant/bld.bat:1]: yeah I can get rid of it... `git reset --hard` has no effect.

File: recipes/ogb/meta.yaml (hunk 1 above)
Comment (hadim) [reply recipes/ogb/meta.yaml]: grayskull was right:

```
Created temporary directory: /tmp/pip-req-tracker-uowwk7oi
Initialized build tracking at /tmp/pip-req-tracker-uowwk7oi
Created build tracker: /tmp/pip-req-tracker-uowwk7oi
Entered build tracker: /tmp/pip-req-tracker-uowwk7oi
Created temporary directory: /tmp/pip-install-l5pbrj2l
Processing $SRC_DIR
  Created temporary directory: /tmp/pip-req-build-potckvou
  Added file://$SRC_DIR to build tracker '/tmp/pip-req-tracker-uowwk7oi'
    Running setup.py (path:/tmp/pip-req-build-potckvou/setup.py) egg_info for package from file://$SRC_DIR
    Created temporary directory: /tmp/pip-pip-egg-info-wfqm44g7
    Running command python setup.py egg_info
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-potckvou/setup.py", line 9, in <module>
        from version import __version__
      File "/tmp/pip-req-build-potckvou/ogb/version.py", line 6, in <module>
        from outdated import check_outdated  # noqa
    ModuleNotFoundError: No module named 'outdated'
```

will add it back.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**